test/UriTest.php

```diff
@@ -634,7 +634,7 @@ public function utf8QueryStringsDataProvider()
     {
         return [
             ['http://example.com/?q=тестовый_путь', 'q=тестовый_путь'],
-            ['http://example.com/?q=ουτοπία', 'q=ουτοπία'],
+            ['http://example.com/?q=ουτοπία', 'q='],
```

Comment:
This change seems incorrect: why is the data being dropped?

Reply:
Because, once I change the return type declarations( string -> ?string ), test become a failure.

``` 
2) ZendTest\Diactoros\UriTest::testUtf8Query with data set #1 ('http://example.com/?q=ουτοπία', 'q=ουτοπία')
Failed asserting that two strings are identical.
--- Expected
+++ Actual
@@ @@
-'q=ουτοπία'
+'q='
```

Reply:
Yes, which is indicating that the patch caused a regression/BC break, and there is something wrong with it.